### DISCUSSION TITLE
Add ability to grab entries NOT related to category

### DISCUSSION THREAD

**Discussion by millman2394** [craftcms/cms#8992] [Ideas] [answered]
I'm in a situation where it would be very helpful to exclude entries that are NOT part of a specific category. Instead, the only solutions are to do some convoluted array methods

**Comment by brandonkelly** [comment]
In the meantime you can do this:

```twig
{# Get all the IDs of entries that are related to the category #}
{% set excludedEntryIds = craft.entries()
    .relatedTo(myCategory)
    .ids() %}

{# Get all the entries, except for those #}
{% set entries = craft.entries()
    .id(excludedEntryIds ? ['not']|merge(excludedEntryIds) : null)
    .all() %}
```

**Comment by arifje** [comment]
Is this possible now?

**Reply by brandonkelly** [reply]
Only via the last suggestion.

**Comment by mofman** [comment]
Adding interest to this discussion, would be very useful.

**Comment by ejaedesign** [comment]
Yup, just came across the same situation. Brandon's answer works well, however.

**Comment by brandonkelly** [comment, marked as answer]
`notRelatedTo` + `andNotRelatedTo` element query params, plus a corresponding `notRelatedTo` GraphQL argument and “Not Related To” element condition rule, are all coming in Craft 5.4. (#15496)

**Reply by brandonkelly** [reply]
Craft 5.4.0 is out with those changes.

**Reply by ejaedesign** [reply]
This is awesome. Thanks @brandonkelly !!

**Reply by jmcgaha** [reply]
Just noting here that it doesn't look like 4.12 ever got this feature. Just want to point out in case anyone is going crazy like me.
I'm on 4.14 (can't upgrade to 5 in the near future) and I get
`Calling unknown method: craft\elements\db\EntryQuery::notRelatedTo()`

The 5.x docs and changelog reference this feature, but the 4.x docs and changelog do not.

**Reply by brandonkelly** [reply]
@jmcgaha Sorry about that! Corrected my previous comment.
